extensions/amp-list/amp-list.md

```diff
@@ -162,6 +162,19 @@ Learn more in [Placeholders & Fallbacks](https://www.ampproject.org/docs/guides/
 </amp-list>
 ```
 
+### Refreshing data
+
+The `amp-list` element exposes a `refresh` action that other elements can reference in `on="tap:..."` attributes. 
```

Comment:
Could you please add this action to the master list of actions and events in https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md

Reply:
Done.